multicast/src/main/kotlin/com/dropbox/flow/multicast/StoreRealActor.kt

```diff
@@ -75,10 +76,15 @@ internal abstract class StoreRealActor<T>(
     }
 
     suspend fun close() {
-        // using a custom token to close so that we can gracefully close the downstream
-        inboundChannel.send(CLOSE_TOKEN)
-        // wait until close is done done
-        closeCompleted.await()
+        try {
+            // using a custom token to close so that we can gracefully close the downstream
+            inboundChannel.send(CLOSE_TOKEN)
+            // wait until close is done done
+            closeCompleted.await()
+        } catch (closed: ClosedSendChannelException) {
```

Comment:
Since `ClosedSendChannelException` is the only exception that can possibly be thrown (as the inboundChannel is closed without `cause`) and we are  ignoring it, should we do this instead?
```kotlin
runCatching {
    // using a custom token to close so that we can gracefully close the downstream
    inboundChannel.send(CLOSE_TOKEN)
    // wait until close is done
    closeCompleted.await()
}
```

Reply:
i dont have a strong opinion here but i personally like the explicit exception catching so that when you read the code, you know what is expected to happen naturally and what not, instead of blindly catching all exceptions. This way, in the future if something changes or there is an unexpected error, we catch it when it happens instead of eating the exception.
What do others think?

Reply:
+1 being explicit